src/ert/gui/model/snapshot.py

```diff
@@ -403,8 +405,10 @@ def _real_data(_index: QModelIndex, node: RealNode, role: int):
 
     @staticmethod
     def _job_data(
-        index: QModelIndex, node: ForwardModelStepNode, role: Qt.ItemDataRole
-    ):
+        index: QModelIndex,
+        node: ForwardModelStepNode,
+        role: int,  # Qt.ItemDataRole
+    ) -> Any:
```

Comment:
Is it a lot of work to get rid of the Any type?

Reply:
there is no point. SnapshotModel.data() is the only function that calls this and it has to return Any due to inheriting QAbstractItemModel.data().